test/unit/utils/rqlQueryParser.test.js

```diff
@@ -0,0 +1,70 @@
+const { expect } = require("chai");
+const { RQLQueryParser, QueryTypes, Operators } = require("../../../utils/RQLParser");
+
+describe("RQLQueryParser", function () {
+  it("should parse filter queries", function () {
+    const queryString = "key:value";
+    const parser = new RQLQueryParser(queryString);
+    const filterQueries = parser.getFilterQueries();
+    expect(filterQueries).to.deep.equal({
+      key: [{ operator: Operators.INCLUDE, value: "value" }],
+    });
+  });
+  it("should parse multiple filter queries", function () {
+    const queryString = "key:value key2:value key:value2";
+    const parser = new RQLQueryParser(queryString);
+    const filterQueries = parser.getFilterQueries();
+    expect(filterQueries).to.deep.equal({
+      key: [
+        { operator: Operators.INCLUDE, value: "value" },
+        { operator: Operators.INCLUDE, value: "value2" },
+      ],
+      key2: [{ operator: Operators.INCLUDE, value: "value" }],
+    });
+  });
+  it("should parse multiple filter with exclude queries", function () {
+    const queryString = "key:value -key2:value -key:value2";
+    const parser = new RQLQueryParser(queryString);
+    const filterQueries = parser.getFilterQueries();
+    expect(filterQueries).to.deep.equal({
+      key: [
+        { operator: Operators.INCLUDE, value: "value" },
+        { operator: Operators.EXCLUDE, value: "value2" },
+      ],
+      key2: [{ operator: Operators.EXCLUDE, value: "value" }],
+    });
+  });
+  it("should parse sort queries", function () {
+    const queryString = "sort:key-desc";
+    const parser = new RQLQueryParser(queryString);
+    const sortQueries = parser.getSortQueries();
+    expect(sortQueries).to.deep.equal({
+      key: "desc",
```

Comment:
In object we expect others properties also 
 {
      operator: Operators.INCLUDE,
      value: value.trim(),
      type: QueryTypes.FILTER,
      key: key.trim(),
    };

Reply:
No check the return type of `getSortQueries`